tests/platform_tests/test_reload_config.py

```diff
@@ -55,3 +58,37 @@ def test_reload_configuration(duthosts, rand_one_dut_hostname, conn_graph_facts,
 
         logging.info("Check sysfs")
         check_sysfs(duthost)
+
+def test_reload_configuration_checks(duthosts, rand_one_dut_hostname, localhost, conn_graph_facts, xcvr_skip_list):
+    """
+    @summary: This test case is to test various system checks in config reload
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+
+    reboot(duthost, localhost, reboot_type=REBOOT_TYPE_COLD, wait=5)
+    logging.info("Reload configuration check")
+    out = duthost.shell("sudo config reload -y", executable="/bin/bash")
+    # config reload command shouldn't work immediately after system reboot
+    assert "Retry later" in out['stdout']
+    assert wait_until(300,20,config_system_checks_passed, duthost)
```

Comment:
missing space 300,20,
update also the same below

Reply:
Done